src/components/tree-view/tree-view.tsx

```diff
@@ -87,7 +98,7 @@ export const TreeItem: React.SFC<TreeItemProps> =
                         {item.label}
                     </span>
                 </div>
-                <div className="nested-tree">
+                {item.children && <ul className="nested-tree" role="group">
                     {state!.isExpanded && (item.children || []).map((child: TreeItemData, index: number) =>
```

Comment:
There is no need to check item.children again here if you are checking it above

Reply:
right